packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/upgrade-range/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,35 @@
+{
+  "dedicatedCloud_datacenters_upgrade_range_title_essentials": "Découvrez votre Managed Bare Metal nouvelle génération",
```

Comment:
Not too found of handling each product traslations in the common component. Ideally in that case, the translation should be injected so later if we need to use it for a third product we don't need to update the component for this purpose